packages/mui-material/src/Tabs/Tabs.js

```diff
@@ -375,9 +375,10 @@ const Tabs = React.forwardRef(function Tabs(inProps, ref) {
     scrollbarWidth: 0,
   });
 
-  const valueToIndex = new Map();
+  const valueToIndex = React.useRef(new Map()).current;
```

Comment:
We use a custom `useLazyRef` for a similar code in Base UI (https://github.com/mui/base-ui/blob/8ba89eadb183833239310328ccf98af2953e17d5/packages/react/src/composite/list/CompositeList.tsx#L19)

Reply:
Used it.